frontend/src/components/SubmitFormDialog.tsx

```diff
@@ -12,23 +13,34 @@ const Wrapper = styled.div`
 interface SubmitFormDialogProps {
     ritm: string
     onConfirmClick: () => void
+    isLoading: boolean
 }
 
-export const SubmitFormDialog = ({ ritm, onConfirmClick }: SubmitFormDialogProps) => {
+export const SubmitFormDialog = ({ ritm, onConfirmClick, isLoading }: SubmitFormDialogProps) => {
     return (
-        <Dialog data-testid={'submit_dialog'}>
-            <Dialog.Title>Order successful!</Dialog.Title>
-            <Dialog.CustomContent>
-                <Typography variant="body_short">
-                    The order was submitted successfully to Service Now. The RITM for your order is
-                </Typography>
-                <Typography variant="body_short_bold">{ritm}</Typography>
-            </Dialog.CustomContent>
-            <Dialog.Actions>
-                <Wrapper>
-                    <Button onClick={onConfirmClick}>OK</Button>
-                </Wrapper>
-            </Dialog.Actions>
-        </Dialog>
+        <>
+            {isLoading ? (
+                <Dialog data-testid={'submit_dialog'}>
+                    <Dialog.CustomContent>
+                        <Spinner centered={true} />
```

Comment:
Consider changing to `<Spinner primary centered={true} />` so the color scheme is consistent. This should be the default value for the Spinner in my opinion

Reply:
Shouldn't we use the fusion default colors when working on a fusion app?

Reply:
Of course it should be primary color.